csharp/src/Google.Protobuf/Reflection/EnumDescriptor.cs

```diff
@@ -112,5 +112,10 @@ public EnumValueDescriptor FindValueByName(string name)
         {
             return File.DescriptorPool.FindSymbol<EnumValueDescriptor>(FullName + "." + name);
         }
+
+        /// <summary>
+        /// The (possibly empty) set of custom options for this enum.
+        /// </summary>
+        public CustomOptions CustomOptions => Proto.Options?.CustomOptions ?? CustomOptions.Empty;
```

Comment:
Can we add Options() method in DescriptorBase, and use extensions to get the the custom options?